lib/manageiq/providers/workflows/builtin_runner.rb

```diff
@@ -0,0 +1,62 @@
+module ManageIQ
+  module Providers
+    module Workflows
+      require "floe"
+
+      class BuiltinRunnner < Floe::Runner
+        SCHEME = "builtin"
+
+        def run_async!(resource, params = {}, secrets = {}, context = {})
+          scheme_prefix = "#{SCHEME}://"
+          raise ArgumentError, "Invalid resource" unless resource&.start_with?(scheme_prefix)
```

Comment:
Not for this PR, but I noticed this pattern is common - wonder if we should push down to the base runner somehow.

Reply:
Or maybe the caller in Floe handles it...so before calling run_async it verifies the scheme with the runner and then gives it the later part with the schem removed?

Reply:
Yeah we could go through the registered schemes and raise an exception if it doesn't match one of them.